tests/foreman/api/test_subscription.py

```diff
@@ -224,8 +218,8 @@ def test_positive_delete_manifest_as_another_user(self):
         assert len(Subscription.list({'organization-id': org.id})) == 0
 
     @pytest.mark.tier2
-    @pytest.mark.libvirt_content_host
     @pytest.mark.usefixtures("golden_ticket_host_setup")
+    @pytest.mark.usefixtures("rhel77_contenthost_class")
```

Comment:
Why is this being marked with `usefixtures` instead of the fixture name being included in the function args?

Reply:
I tried adding it to the functions args but it did not work. I think it's because the test is within a class. So I used 'usefixtures' to reference it within self.

Reply:
let's dig deeper at why it didn't work, the fact that the test is in a class shouldn't preclude it from including fixtures that operate at any scope.

Reply:
That being said, the fixture markers could be corrected separately from these tests. If @JacobCallahan comments have been addressed to his satisfaction, I am happy to otherwise approve this PR. As the use of `APITestCase` is removed from the modules, we can correct how fixtures are added to the tests.

Reply:
I'd like to see this approach attempted.

Reply:
To resolve this thread and respond to @ColeHiggins2 comment on the PR itself - this is due to the test's class being based on `APITestCase`. This should be resolved as part of the pytest conversion of the class.